keyboards/dz60/keymaps/mekberg/keymap.c

```diff
@@ -0,0 +1,102 @@
+#include QMK_KEYBOARD_H
+
+/*
+DZ60 ISO physical layout
+1u == 8chars
+,-------------------------------------------------------------------------------------------------------------------------------------.
+|        |        |        |        |        |        |        |        |        |        |        |        |        |      2u        |
+|-------------------------------------------------------------------------------------------------------------------------------------|
+|    1,5u    |        |        |        |        |        |        |        |        |        |        |        |        |     1,5u   |
+|--------------------------------------------------------------------------------------------------------------------------|          |
+|     1,75u    |        |        |        |        |        |        |        |        |        |        |        |        |   1,25u  |
+|-------------------------------------------------------------------------------------------------------------------------------------|
+|   1,25u  |        |        |        |        |        |        |        |        |        |        |        |         2,75u         |
+|-------------------------------------------------------------------------------------------------------------------------------------|
+|   1,25u  |   1,25u  |   1,25u  |                                                        |   1,25u  |   1,25u  |   1,25u  |   1,25u  |
+`-------------------------------------------------------------------------------------------------------------------------------------'
+*/
+
+// Fillers to make layering more clear
+#define XXXXXX KC_TRNS
+#define ______ KC_NO
+
+// ISO keycodes with Swedish layout names
+#define SE_PLUS KC_MINS         // +
+#define SE_TCK KC_EQL           // Fronttick (´)
+#define SE_DASH KC_SLSH         // -
+#define SE_UML KC_RBRC          // Umlaut (¨)
+#define SE_QUT KC_NUHS          // Singlequote (')
+#define SE_AUML KC_QUOT     	// Ä
+#define SE_ARNG KC_LBRC     	// Å
+#define SE_OUML KC_SCLN     	// Ö
+#define SE_LT KC_NUBS       	// <
+
+// My common shortcuts
+#define MY_PREV KC_MRWD         // Media: Previous
+#define MY_NEXT KC_MFFD         // Media: Next
+#define MY_PLAY KC_MPLY         // Media: Play/Pause
+#define MY_VOLU KC__VOLUP       // Media: Volume Up
+#define MY_VOLD KC__VOLDOWN     // Media: Volume Down
+#define MY_MUTE KC__MUTE        // Media: Mute
+#define MY_SLEP G(A(KC_POWER))  // Mac: Sleep
```

Comment:
You might want to use the media keycodes for these, as they are more widely recognised across OSes:
https://docs.qmk.fm/#/keycodes_basic?id=media-keys

Reply:
I think I found these as Mac suitable somewhere, but I'll test it and see if it works, and change it if it does.

Reply:
Closing this comment, I haven't received my dz60 yet so keeping it out until I can actually test things myself.

Reply:
I can tell you for sure that `KC_VOLU`, `KC_VOLD`, `KC_MUTE`, and `KC_PWR` all work fine on macOS ;)

Reply:
Yeah, I tested it quickly on my Let's Split and it does indeed work. Committed as fixed already :)